iceoryx_hoofs/platform/qnx/include/iceoryx_hoofs/platform/platform_settings.hpp

```diff
@@ -32,7 +32,7 @@ constexpr uint64_t IOX_MAX_PATH_LENGTH = 1023U;
 constexpr bool IOX_SHM_WRITE_ZEROS_ON_CREATION = true;
 constexpr uint64_t IOX_MAX_SHM_NAME_LENGTH = 1024U;
 constexpr const char IOX_PATH_SEPARATORS[] = "/";
-constexpr uint64_t IOX_UDS_SOCKET_MAX_MESSAGE_SIZE = 4096;
+constexpr uint64_t IOX_UDS_SOCKET_MAX_MESSAGE_SIZE = 3990;
```

Comment:
This is a weird number. Is this QNX specific or is it defined for QNX on a specific target?